NickvisionTubeConverter.Shared/Resources/Strings.de.resx

```diff
@@ -334,7 +334,7 @@ marcin https://github.com/martin-desktops</value>
     <value>Wenn diese Option aktiviert ist, werden die Video-Metadaten in die heruntergeladene Datei aufgenommen.</value>
   </data>
   <data name="VideoUrl.Invalid" xml:space="preserve">
-    <value>Video Url (ungültig)</value>
+    <value>Video URL (ungültig)</value>
```

Comment:
You don't need to change anything in Strings.*.resx, Weblate will look for all changes in original Strings.resx and make all translation files match.